airflow-core/tests/unit/cli/commands/test_dag_command.py

```diff
@@ -55,8 +54,14 @@
     clear_db_runs,
     parse_and_sync_to_db,
 )
+from tests_common.test_utils.version_compat import AIRFLOW_V_3_1_PLUS
 from unit.models import TEST_DAGS_FOLDER
 
+if AIRFLOW_V_3_1_PLUS:
+    from airflow.sdk.bases.operator import BaseOperator
```

Comment:
```suggestion
    from airflow.sdk import BaseOperator
```

As a rule of thumb, `airflow.sdk` is the public interface of the sdk and should be used instead of modules inside whenever possible.

Reply:
Also should this be 3.0+ instead?